code/datums/components/surgery_initiator.dm

```diff
@@ -310,3 +348,50 @@
 
 /datum/component/surgery_initiator/robo/sharp
 	valid_starting_types = SURGERY_INITIATOR_ORGANIC | SURGERY_INITIATOR_ROBOTIC
+
+/datum/component/surgery_initiator/cloth
+	can_cancel = FALSE
+	surgery_start_sound = "rustle"
+
+/datum/component/surgery_initiator/cloth/show_starting_message(mob/user, mob/living/target, datum/surgery/procedure)
+	user.visible_message(
+		"<span class='notice'>[user] drapes [parent] over [target]'s [parse_zone(user.zone_selected)] to prepare for surgery.</span>",
+		"<span class='notice'>You drape [parent] over [target]'s [parse_zone(user.zone_selected)] to prepare for \an [procedure.name].</span>",
+	)
+
+/datum/component/surgery_initiator/cloth/try_choose_surgery(mob/user, mob/living/target, datum/surgery/surgery)
+	var/datum/surgery/new_procedure = ..()
+	if(!istype(new_procedure))
+		return
+
+	new_procedure.started_with_drapes = TRUE
+
+/datum/component/surgery_initiator/cloth/on_surgery_selection(mob/user, mob/living/target, datum/surgery/target_surgery)
+	user.visible_message(
+		"<span class='notice'>[user] starts to apply [parent] onto [target].</span>",
+		"<span class='notice'>You start to apply [parent] onto [target].</span>",
+	)
+
+	if(!isnull(surgery_start_sound))
+		playsound(src, surgery_start_sound, 50, TRUE, FALSE)
```

Comment:
Not sure FALSE is a valid argument here for this parameter. 
Rather, it should probably just be 0, since it is looking for a number to modify the range with.